src/opnsense/mvc/app/models/OPNsense/Dnsmasq/Dnsmasq.php

```diff
@@ -259,6 +260,26 @@ public function performValidation($validateFullModel = false)
                     )
                 );
             }
+
+            if (
+                !$option->value->isEmpty() &&
+                !$option->option6->isEmpty()
+            ) {
+                $values = array_map('trim', explode(',', (string)$option->value));
+                foreach ($values as $value) {
+                    if (
+                        Util::isIpv6Address(trim($value, '[]')) &&
```

Comment:
[[fe80::1]] probably doesn't validate so that's why I suggested to avoid the trim and just check for IPv6 ;)